openreview/venue_request/venue_request.py

```diff
@@ -746,10 +746,16 @@ def setup_submission_revision_stage(self):
             },
             'submission_revision_deadline': {
                 'description': 'By when should the authors finish revising submissions? Please enter a time and date in GMT using the following format: YYYY/MM/DD HH:MM (e.g. 2019/01/31 23:59) (Skip this if your venue does not have submission revisions)',
-                'value-regex': r'^[0-9]{4}\/([1-9]|0[1-9]|1[0-2])\/([1-9]|0[1-9]|[1-2][0-9]|3[0-1])(\s+)?((2[0-3]|[01][0-9]|[0-9]):[0-5][0-9])?(\s+)?$',
-                'required': True,
+                'value-regex': r'(^$)|(^[0-9]{4}\/([1-9]|0[1-9]|1[0-2])\/([1-9]|0[1-9]|[1-2][0-9]|3[0-1])(\s+)?((2[0-3]|[01][0-9]|[0-9]):[0-5][0-9])?(\s+)?$)',
```

Comment:
Let's keep duedate optional and expdate mandatory so no need to add a validation in the pre process.

All invitations should start having expiration dates otherwise they could be active forever leading to date process and process functions errors and keeping space in our queue.